src/main/java/org/libsodium/jni/Sodium.java

```diff
@@ -737,10 +737,6 @@ public static int crypto_onetimeauth_poly1305_verify(byte[] h, byte[] in, int in
     return SodiumJNI.crypto_onetimeauth_poly1305_verify(h, in, inlen, k);
   }
 
-  public static int crypto_onetimeauth_poly1305_statebytes() {
```

Comment:
Could be a different version. Maybe try running 'submodule-update.sh' to update the libsodium version? That will bring to the latest version of branch 'stable'